Shared/Samples/Create load report/CreateLoadReportView.Model.swift

```diff
@@ -15,6 +15,28 @@
 import ArcGIS
 import SwiftUI
 
+extension CreateLoadReportView {
+    fileprivate struct PhaseSummaries {
```

Comment:
Can you please document this type and its members? Also you may want to move it to the same extension as `PhaseSummary`.  I just placed it here in my branch to quickly prototype.